script/data.js

```diff
@@ -0,0 +1,143 @@
+    description2:
+      'A daily selection of privately personalized; no accounts or sign-ups required.',
+    company2: 'CONOPY',
+    no: 'detail1',
+    cardImage1: './images/Mobpics2.png',
+    counterText: 'Back End Dev',
+    year: '2015',
+    technologies: ['HTML', 'CSS', 'JavaScript'],
+  },
+
+  {
+    id: '3',
+    cardTitle: 'TONIC',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    company: 'CANOPY',
+    cardTitle2: 'TONIC',
+    description2:
+      'A daily selection of privately personalized; no accounts or sign-ups required.',
+    company2: 'CONOPY',
+    no: 'detail1',
+    cardImage1: './images/Mobpics3.png',
+    counterText: 'Back End Dev',
+    year: '2015',
+    technologies: ['HTML', 'CSS', 'JavaScript'],
+  },
+
+  {
+    id: '4',
+    cardTitle: 'Multi-Post Stories',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    company: 'CANOPY',
+    cardTitle2: 'TONIC',
+    description2:
+      'A daily selection of privately personalized; no accounts or sign-ups required.',
+    company2: 'CONOPY',
+    no: 'detail1',
+    cardImage1: './images/Mobpics4.png',
+    counterText: 'Back End Dev',
+    year: '2015',
+    technologies: ['HTML', 'CSS', 'JavaScript'],
+  },
+];
+
+// Desktop Version//
+export const desktopProjects = [
+  {
+    // card 1 details
+    id: '1',
+    cardTitle: 'TONIC',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    company: 'CANOPY',
+    cardTitle2: 'TONIC',
+    description2:
+      'A daily selection of privately personalized; no accounts or sign-ups required.',
+    company2: 'CONOPY',
+    no: 'detail1',
+    cardImage1: './images/modal-image.png',
+    counterText: 'Back End Dev',
+    year: '2015',
+    technologies: ['HTML', 'CSS', 'JavaScript'],
+  },
+
+  {
+    id: '2',
+    cardTitle: 'Multi-Post Stories',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    company: 'FACEBOOK',
+    cardTitle2: 'TONIC',
+    description2:
+      'A daily selection of privately personalized; no accounts or sign-ups required.',
+    company2: 'CONOPY',
+    no: 'detail1',
+    cardImage1: './images/desktop2.png',
+    counterText: 'Full Stack Dev',
+    year: '2015',
+    technologies: ['HTML', 'CSS', 'JavaScript'],
+  },
+
+  {
+    id: '3',
+    cardTitle: 'TONIC',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    company: 'FACEBOOK',
+    cardTitle2: 'Facebook 360',
+    description2:
+      'A daily selection of privately personalized; no accounts or sign-ups required.',
+    company2: 'CONOPY',
+    no: 'detail1',
+    cardImage1: './images/desktop3.png',
+    counterText: 'Full Stack Dev',
+    year: '2015',
+    technologies: ['HTML', 'CSS', 'JavaScript'],
+  },
```

Comment:
[OPTIONAL] 
- [ ] Please consider adding at least 4 to 5 line text for your project description to make your work more professional.